docs/source/matlab/common.rst

```diff
@@ -0,0 +1,83 @@
++-----------------+----------------------+------------------------------------------------------------------------+
+| ``not``         | ``logical_not``      | ``X.logical_not(Y)``                                                   |
++-----------------+----------------------+------------------------------------------------------------------------+
+| ``or``          | ``logical_or``       | ``X.logical_or(Y)``                                                    |
++-----------------+----------------------+------------------------------------------------------------------------+
+| ``plus``        | ``__add__``          | ``X + Y``                                                              |
++-----------------+----------------------+------------------------------------------------------------------------+
+| ``power``       | ``__pow__``          | ``X ** a``                                                             |
++-----------------+----------------------+------------------------------------------------------------------------+
+| ``rdivide``     | ``__rtruediv__``     | ``X / Y``                                                              |
++-----------------+----------------------+------------------------------------------------------------------------+
+| ``subsasgn``    | ``__setitem__``      | ``X[index] = a``                                                       |
++-----------------+----------------------+------------------------------------------------------------------------+
+| ``subsref``     | ``__getitem__``      | ``X[index]``                                                           |
++-----------------+----------------------+------------------------------------------------------------------------+
+| ``tenfun``      | ``tt_tenfun``        | e.g., ``pyttb.tt_tenfun(lambda x: x + 1, A)``                          |
++-----------------+----------------------+------------------------------------------------------------------------+
+| ``times``       | ``__mul__``          | ``X * Y``                                                              |
++-----------------+----------------------+------------------------------------------------------------------------+
+| ``uminus``      | ``__neg__``          | ``-X``                                                                 |
++-----------------+----------------------+------------------------------------------------------------------------+
+| ``uplus``       | ``__pos__``          | ``+X``                                                                 |
++-----------------+----------------------+------------------------------------------------------------------------+
+| ``xor``         | ``logical_xor``      | ``X.logical_xor(Y)``                                                   |
++-----------------+----------------------+------------------------------------------------------------------------+
+
+Copying
```

Comment:
It might be useful to note this is fairly common for non-builtin types in python and include a link. Hereish? https://docs.python.org/3/library/copy.html